### PR TITLE
Align vscode DevDependency for extension projects

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
The dependency is needed for the testrunner.js that's included in the package to run unit tests. Starting with 1.1.32 though the package now includes the vscode.d.ts typings file - which then conflicts with our own checked in version. Some further discussion about how to handle this is needed but for the time being I'm aligning all of our extension projects to at least use the same package version so that we don't have different extensions potentially running tests differently.

### CODE REVIEW

File: extensions/admin-tool-ext-win/package.json [hunk 1]
```diff
@@ -70,6 +70,6 @@
 		"vscode-nls": "^3.2.1"
 	},
 	"devDependencies": {
-		"vscode": "1.0.1"
+		"vscode": "^1.1.33"
```
Comment (alanrenmsft) [inline extensions/admin-tool-ext-win/package.json]: do you mean 1.33.1?

Comment (Charles-Gagnon) [reply extensions/admin-tool-ext-win/package.json]: No, the npm package version is different than vscode version. The latest npm package is 1.1.33.

**Comment by anthonydresser** [comment]
Not sure if it will help here and with @kevcunnane but vscode has a entirely separate package for vscode testing.
https://github.com/Microsoft/vscode-test
https://www.npmjs.com/package/vscode-test

that you can pull in independently of the typings. the vscode-extension-vscode github project (npm vscode) just pulls in this package as a dependency.

**Comment by anthonydresser** [comment]
It might be simpler to not refer to the vscode npm pacakge and instead refer directly to our .d.ts and just pull in the vscode-test package for the test runner.

**Comment by Charles-Gagnon** [comment]
@anthonydresser The goal here is just to align the version of the test runner we were using, it's mostly a consistency thing. It started off as me wanting to make sure that this extension wasn't stuck using a really old version of the vscode package and then I figured I'd end up with at least making all our extensions use the same package until we could figure out a way to get the testrunner outside of the vscode package.

As a side note - I didn't update to the latest vscode package because starting with 1.1.32 they're including vscode.d.ts in that package - which then conflicts with our vscode.d.ts (duplicate definitions).

I looked at the vscode-test but it looks like it still either requires :

1. You use the vscode testrunner - https://github.com/octref/vscode-test-sample/blob/master/test/suite/index.ts
2. You implement you own testrunner (just a run method that calls running the tests)

So what I might do is just take their testrunner and copy it into our src directly since it's a pretty generic function. Then we can get rid of the vscode requirement directly.

**Comment by Charles-Gagnon** [comment]
Closing this for now as I'm going to investigate using the testrunner Kevin mentioned

**Pull request closed without merging**